habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/RetrieveWheelsMojo.java

```diff
@@ -32,6 +32,11 @@ public class RetrieveWheelsMojo extends AbstractHabushuMojo {
     @Parameter(property = "habushu.wheelDependencies", required = false)
     protected List<WheelDependency> wheelDependencies;
 
+    @Override
+    public void execute() throws MojoExecutionException, MojoFailureException {
```

Comment:
A: Probably worth clarifying the logic being skipped in a comment.

Reply:
@cpointe-ibllanos added.